src/Microsoft.DotNet.Interactive.Tests/LanguageKernelFormattingTests.cs

```diff
@@ -26,6 +27,18 @@ public LanguageKernelFormattingTests(ITestOutputHelper output) : base(output)
         {
         }
 
+        protected override CSharpKernel CreateCSharpKernel()
+        {
+            var cSharpKernel = base.CreateCSharpKernel();
+
+            cSharpKernel.DeferCommand(new SubmitCode(@$"
```

Comment:
```suggestion
            cSharpKernel.DeferCommand(new SubmitCode($@"
```

Both types of string are technically allowed, but `$@` is the canonical way to do it.